sdk/servicebus/service-bus/package.json

```diff
@@ -87,24 +100,26 @@
     "build:browser": "tsc -p . && cross-env ONLY_BROWSER=true rollup -c 2>&1",
     "build:node": "tsc -p . && cross-env ONLY_NODE=true rollup -c 2>&1",
     "build:samples": "node .scripts/prepSamples.js && cd samples && tsc -p .",
-    "build:test": "tsc -p . && cross-env ONLY_NODE=true rollup -c rollup.test.config.js 2>&1",
-    "build": "tsc -p . && rollup -c 2>&1",
+    "build:test:browser": "tsc -p . && cross-env BROWSER_TEST=true rollup -c 2>&1",
+    "build:test:node": "tsc -p . && cross-env ONLY_NODE=true rollup -c rollup.test.config.js 2>&1",
+    "build:test": "npm run build:test:node && npm run build:test:browser",
+    "build": "npm run build:node && npm run build:browser && npm run build:test:node && npm run build:test:browser",
```

Comment:
build shouldn't call "build:test" - the build server will call both of those tasks as needed.

Reply:
Honestly the best thing to do here is just to leave `build` as it was. That way it only runs `tsc` once:

`"build": "tsc -p . && rollup -c 2>&1",`

Reply:
Reverted, but I'm a little doubtful if this would work as build steps for node and browser seem different?